index.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio-website</title>
+  </head>
+  <body>
+    <header>
+      <img src=".png\geometry_1.png" />
+      <button class="menu">
+        <img src=".png\ic_menu.png" />
+      </button>
+    </header>
+    <section>
+      <h1>Abdelaziz Mohamed</h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Taka a look of my works. If you like what you see
+        and have a project you need coded, don’t hesitate and contact me.
+      </p>
+      <button class="btn">Start Collaboration</button>
+    </section>
```

Comment:
Kindly take into account that your section should also contain this background image 😉:
![image](https://user-images.githubusercontent.com/42799579/198755354-7e95e8c2-4395-449e-9136-e6a91d0aec08.png).